package.json

```diff
@@ -53,6 +53,11 @@
     "react-i18next": "^11.8.13"
   },
   "dependencies": {
-    "react-number-format": "^4.5.5"
+    "@date-io/date-fns": "^1.3.13",
+    "@material-ui/pickers": "^3.3.10",
+    "add": "^2.0.6",
+    "date-fns": "^2.22.1",
+    "react-number-format": "^4.5.5",
+    "yarn": "^1.22.10"
```

Comment:
There might be an error here, I don't think we need the dependencies "add" and "yarn" :)
Note: they must be removed from the yarn.lock file too

Reply:
Ah, I thought so! but maybe it's because I didn't use the latest version of `@date-io/date-fns`?

Reply:
A typo in a command is very likely the cause of these extra dependencies: `yarn add yarn add foo` will add dependencies to "yarn", "add" and "foo".

Reply:
I just realized that date-fns is added twice, this must be fixed too, indeed :)